### PR TITLE
VoiceOver accessibility improvements

### PR DESCRIPTION
**Pull request by adincebic**
- Make CategoryCollectionViewCell accessible for voiceover.
- Add support for accessibility Z gesture in CategoryDetailViewController.

### CODE REVIEW

File: swift_algorithms.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -935,7 +935,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
-				DEVELOPMENT_TEAM = 9L49SS68XP;
+				DEVELOPMENT_TEAM = SD6G3DF8CS;
 				INFOPLIST_FILE = swift_algorithms/Info.plist;
```
Comment (thexande) [inline swift_algorithms.xcodeproj/project.pbxproj]: @adincebic Would you mind reverting your project file change here?

**Review by thexande — COMMENTED** [review]
@adincebic Thanks so much for contributing! Would you mind reverting the project file and we can merge? Thanks!!!

File: swift_algorithms.xcodeproj/project.pbxproj (hunk 1 above)
Comment (adincebic) [reply swift_algorithms.xcodeproj/project.pbxproj]: Thanks for your response. I reverted the project file.

Comment (adincebic) [reply swift_algorithms.xcodeproj/project.pbxproj]: @thexande can we merge this now?

**Comment by thexande** [comment]
@adincebic Well done. Thanks for the contribution.

**Pull request merged**